modules/core/src/main/java/org/apache/ignite/internal/util/future/GridFutureAdapter.java

```diff
@@ -53,6 +57,10 @@
     private static final AtomicReferenceFieldUpdater<GridFutureAdapter, Object> stateUpdater =
         AtomicReferenceFieldUpdater.newUpdater(GridFutureAdapter.class, Object.class, "state");
 
+    /** Tracing span. */
+    @GridToStringExclude
+    protected volatile Span span;
```

Comment:
Good decision.
Seems like we can remove field existing "span" field from TX futures.

Reply:
Fixed.